setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 99
```

Comment:
Are we sure?

Reply:
My personal opinion is that restricting line lengths is stupid because line wrap exists, so why hard code when the line ends?
It's basically just a nuisance when writing code because you have to wrap and often make code harder to read which is the opposite of what flake8 wants you to achieve. This person expressed it pretty well, and in fact uses 119 like django (a huge project) https://simpleisbetterthancomplex.com/packages/2016/08/05/flake8.html

Reply:
Linewrap in Python is more dangerous than semicolon using languages :-)

I'm not averse to increasing the minimum (you'll see that some of my lines have `# noqa`), and I see that the Python SDK pushes it to 119. I'd keep to 99 for now.